rpcserver.go

```diff
@@ -4168,11 +4168,9 @@ func createRPCOpenChannel(r *rpcServer, dbChannel *channeldb.OpenChannel,
 	// Look up our channel peer's node alias if the caller requests it.
 	if peerAliasLookup {
 		peerAlias, err := r.server.graphDB.LookupAlias(nodePub)
-		if err != nil {
-			return nil, fmt.Errorf("unable to lookup peer "+
```

Comment:
I think the error needs to be logged. If it's an expected behavior, we can use `rpcsLog.Debugf` otherwise `rpcsLog.Errorf`.

Reply:
I argued against logging the error. Since this seems to be a not too uncommon occurrence, this would spam the log. See https://github.com/lightningnetwork/lnd/pull/7322#issuecomment-1450328599.

Reply:
yeah was thinking about debugging, not familiar enough tho, just think a returned error should be caught.